make/node.mk

```diff
@@ -79,3 +79,35 @@ node-coffee-edit-check:
 node-update-makefile:
 	@wget https://raw.githubusercontent.com/Clever/dev-handbook/master/make/node.mk -O /tmp/node.mk 2>/dev/null
 	@if ! grep -q $(NODE_MK_VERSION) /tmp/node.mk; then cp /tmp/node.mk node.mk && echo "node.mk updated"; else echo "node.mk is up-to-date"; fi
+
+##########################
+## Linting / Formatting ##
+##########################
+# Match .js, .ts, .jsx, and .tsx files. Ignore directories contaning installed dependencies or auto-generated files.
+NODE_ALL_FILES := $(shell find . -regex '.*\.[jt]s[x]\{0,1\}' -not -path "./node_modules/*" -a -not -path "./gen-js/*" -a -not -path "./build/*" -a -not -path "./__build/*")
```

Comment:
Yeah. While I'm pretty sure the regex is right, I'd almost prefer (ts|js|tsx|jsx) for clarity.